app/demo-interactor/HostKNDemoRunner.cc

```diff
@@ -0,0 +1,213 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file HostKNDemoRunner.cc
+//---------------------------------------------------------------------------//
+#include "HostKNDemoRunner.hh"
+
+#include <iostream>
+#include <random>
+#include "base/Stopwatch.hh"
+#include "base/Range.hh"
+#include "base/ArrayUtils.hh"
+#include "random/distributions/ExponentialDistribution.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/Units.hh"
+#include "physics/base/Secondary.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/em/KleinNishinaInteractor.hh"
+#include "PhysicsArrayCalculator.hh"
+#include "DetectorView.hh"
+#include "HostStackAllocatorStore.hh"
+#include "HostDetectorStore.hh"
+
+namespace demo_interactor_cpu
```

Comment:
You're welcome to share the `demo_interactor` namespace 😉

Reply:
I originally had the names of the classes the same, then I changed them. I'll move the namespace back to `demo_interactor`